Oregon Nexus - Freshman On Track PoweBI Dashboard/readme.md

```diff
@@ -0,0 +1,21 @@
+# Contribution Template
+
+
+## Brief Description
+A series of views, tables and procedure to help with denormalize and aggregate the data for use with the Freshmen On Truck PowerBI Dashboard. 
+
+## Support ODS/AMT Versions
+
+* ODS 3.1.1 
+* AMT v2 (March 2020)
+
+## Assumptions
+We were looking for a quick and easy way to build a PowerBI Dashboard without moving the data out of the production ODS. 
+With the help of AMT which provided the base for our project, we built a series of views to aggregate the data we needed. The stored procedure uses the view to populate the staging tables that are then used by the PowerBI Dashboard.
+The idea was that the procedure can be scheduled to run nighly.
+
```

Comment:
It appears that you are building an at Risk GPA out of  core subjects.  Can you add a short description describing which classes make up the at risk GPA and how it is defined?